### PR TITLE
Feat modal

### PR DESCRIPTION
**Pull request by abhij1607**
image
### feat- modal

### CODE REVIEW

File: components/modal.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.modal.fade {
+  z-index: 3;
+  display: none;
+  padding-top: 100px;
```
Comment (saadahmadhere) [inline components/modal.css]: relative units could be used here.

File: docs/modal.html [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                  <button id="modal-close" class="modal-close">x</button>
+                  <h3>Heading</h3>
+                  <p>line1</p>
+                  <p>line2</p>
+                </div>
+              </div>
+            </div>
+            <button class="btn btn-primary" id="modal-show">show modal</button>
+          </div>
+          <script>
```
Comment (saadahmadhere) [inline docs/modal.html]: The code would be cleaner if the JS was used in an external file.

**Review by saadahmadhere — COMMENTED** [review]
Decently written code, good use of utility classes. Aesthetic.

**Review by MaruthiPothuganti — COMMENTED** [review]
Aside Nav isnt updated for all the new components. Everything else looks good.

File: components/modal.css (hunk 1 above)
Comment (abhij1607) [reply components/modal.css]: thanks for pointing it out, will update it

**Comment by abhij1607** [comment]
merging this PR

**Pull request merged**